apps/services/schemas.py

```diff
@@ -90,8 +91,19 @@ def validate_status_id(cls, v):
 
 
 class UserRequestReviewOut(BaseOutSchema):
-    status_id: int
     user_request_review_id: int
+    created_at: datetime
```

Comment:
more of field can be empty
please double check

Reply:
Checked, everything looks good